### ISSUE TITLE
Feature request: background image

### ISSUE THREAD

**Issue by ambipur76** [prasathmani/tinyfilemanager#223]
Is it possible to add a background image to the login page an the file lister page? A new variable would be needed to adjust the path of the background image.

**Comment by EV-soft** [comment]
Is this what you want to achieve?
`https://www.sky-box.dk/tinyFM/TFM-v2.3.8c.php`

**Comment by ambipur76** [comment]
I suppose your background image is a small repetitive on, but if this is possible, than setting one big as fixed is also possible. Setting at least 1280 * 1024 size and also for the file lister page would be the best. Thank's a lot! Where did you change the php code?

**Comment by EV-soft** [comment]
Login-page:
```
SEARCH: body.fm-login-page{background-color:#f7f9fb;font-size:14px}
REPLACE:  body.fm-login-page{background-color:#f7f9fb;background-image: url('.FM_background.png');font-size:14px}
```
File-view-table:
```
SEARCH: #main-table span.badge:nth-child(6){border-color:#45c3d2}
REPLACE: #main-table span.badge:nth-child(6){border-color:#45c3d2} #main-table {background-image:url('.FM_background.png');}
```
The file FM_background.png should be placed in the same directory as tinyfilemanager.php

**Comment by ambipur76** [comment]
Thanks EV-soft! I have applied background-repeat: no-repeat; background-attachment: fixed; because my background image is a fullscreen -size. I have also added the background image to the body section ow the filelister page:
        body {
            font-size: 14px;
            color: #222;
            background: #F7F7F7;
            background-image: url('background.jpg');
            background-repeat: no-repeat;
            background-attachment: fixed;
        }
This way the background image is also present outside the filelist table. I small thing still missing: in the bottom bar (Full Size: 4.07 MB Files: 15 Folders: 4 Memory used: 2 MB Partition size: 18.2 GB free of 578.54 GB) the background is a white background around the displayed numbers however I would also expect here my background image.
